source/how-tos/boot-micropython.rst

```diff
@@ -0,0 +1,110 @@
+.. _boot-micropython:
+
+Boot MicroPython
+-----------------
+
+|MicroPython| is a lean and efficient implementation of the Python 3 programming language
+that includes a small subset of the Python standard library. I can be used to run python
```

Comment:
Can we add a line - 

|MicroPython| is a lean and efficient implementation of the Python 3 programming language
that includes a small subset of the Python standard library. MicroPython can be built as a standalone module that can be launched by SBL's OsLoader payload. This can then be used to run python based applications.

Reply:
Good point. Let me add that.

Reply:
Sounds good.